frontend/angular/README.md

```diff
@@ -0,0 +1,20 @@
+## Introduction
+Angular is a popular front-end framework made by Google. Like other popular front-end frameworks, it uses a component-based architecture to structure apps.
+
+If you are a beginner for Angular and wonder how to start, This article is for you and we focus only on basic Angular App with one page. Once finished our app looks like below where we can add items, edit, delete items.
```

Comment:
```suggestion
If you are a beginner for Angular and wonder how to start, this article is for you and we focus only on basic Angular App with one page. Once finished our app looks like below where we can add items, edit, delete items.
```

Which finished app are you talking about?

Reply:
I mixed something else here. I have rectified it. Please review, sir.